src/pages/Edit/MessageEditPage.jsx

```diff
@@ -1,44 +1,66 @@
 import styled from 'styled-components';
-import { useState } from 'react';
+import { useState, useEffect } from 'react';
 import BackHeaderComponent from '../../components/common/BackHeaderComponent';
 import ButtonComponent from '../../components/common/ButtonComponent';
 import BottomBackgroundComponent from '../../components/common/BottomBackgroundComponent';
-const tempList = [
-  {
-    name: '이름은최대8글자',
-    message:
-      '축하 메시지가 표시될 거예요 메시지는 여러 줄이어도 모두 표시되도록 설정해 주세요',
-    price: '40,000원',
-  },
-  {
-    name: '김이화',
-    message: '생일 축하해!',
-  },
-  {
-    name: '이름은최대8글자',
-    message:
-      '축하 메시지가 표시될 거예요 메시지는 여러 줄이어도 모두 표시되도록 설정해 주세요',
-  },
-];
-const Btn = (
-  <ButtonComponent
-    btnInfo={{
-      text: '수정하기',
-      width: '335px',
-      color: 'orange',
-    }}
-  />
-);
+import { getCongratsMessage, patchCongratsMessage } from '../../api/funding';
+import { useNavigate, useParams } from 'react-router-dom';
+
 const MessageEditPage = () => {
-  const [MsgText, setMsgText] = useState(tempList[0].message);
+  const participationId = 19; //아직 연결 안함 본인 participationId 넣어야 작동됨
+  const { fundingId } = useParams();
+  const [anony, setAnony] = useState();
+  const [msgText, setMsgText] = useState('');
+  const [name, setName] = useState('nickname');
   const handleInputChange = (e) => {
     setMsgText(e.target.value);
   };
   const handleRadioChange = (e) => {
-    setName(e.target.value);
+    const selectedValue = e.target.value;
+    setName(selectedValue);
+    setAnony(selectedValue === 'anony' ? true : false);
   };
-  const [name, setName] = useState('nickname');
-
+  const readMessage = async (fundingId) => {
+    try {
+      const response = await getCongratsMessage(fundingId);
+      const isAnonymous = response.data.anonymity === true;
+      setMsgText(response.data.message);
+      setName(isAnonymous ? 'anony' : 'nickname');
```

Comment:
서버가 보내주는 `anonymity` 데이터가 불린 타입이라 바로 사용해도 될 것 같아요!
```suggestion
      const response = await getCongratsMessage(fundingId);
      setMsgText(response.data.message);
      setName(response.data.anonymity ? 'anony' : 'nickname');
```

Reply:
수정했습니다